src/server/scripts/EasternKingdoms/BlackrockMountain/BlackrockDepths/blackrock_depths.cpp

```diff
@@ -119,13 +153,18 @@ class at_ring_of_law : public AreaTriggerScript
     {
         if (InstanceScript* instance = player->GetInstanceScript())
         {
+            time_t now = time(nullptr);
```

Comment:
what is this being used for?

Reply:
Warding off so it gives 2 minutes between each wipe (yeah somewhat in magic numbers) on line 161. There's probably a better way than that. I couldn't figure if the event today had any kind of reset.